src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/DefaultAssemblyProvider.cs

```diff
@@ -88,7 +88,7 @@ public IEnumerable<Assembly> CandidateAssemblies
         /// </summary>
         /// <returns>A set of <see cref="Library"/>.</returns>
         // Internal for unit testing
-        protected internal virtual IEnumerable<RuntimeLibrary> GetCandidateLibraries()
```

Comment:
Why remove protected? It wasn't consumed in the tests that way previously. It was always called via the internal accessor method.